manage_breast_screening/mammograms/jinja2/mammograms/record_medical_information.jinja

```diff
@@ -1,105 +1,84 @@
 {% extends "wizard_step.jinja" %}
 {% from "nhsuk/components/button/macro.jinja" import button %}
-{% from "nhsuk/components/card/macro.jinja" import card %}
 {% from "nhsuk/components/inset-text/macro.jinja" import insetText %}
 {% from "nhsuk/components/summary-list/macro.jinja" import summaryList %}
-{% from 'nhsuk/components/tag/macro.jinja' import tag %}
 {% from "components/participant-details/summary_list_rows.jinja" import last_mammogram_html %}
+{% from "mammograms/components/card_with_status.jinja" import card_with_status %}
 
 {% block step_content %}
 
-<div id="{{ presenter.get_anchor(sections.MAMMOGRAM_HISTORY) }}" class="app-reviewable-section">
-  {% call card({"heading": "Mammogram history"}) %}
-    <div class="app-reviewable-section__status">
-      {{ tag(presenter.get_section_tag_properties(sections.MAMMOGRAM_HISTORY)) }}
-    </div>
+{% call card_with_status(
```

Comment:
🤔 I think the presenter for this page is also getting quite big. Lets see how it looks after the medical history is done, but maybe a future refactor would be to have a presenter for each section, which would group together the section name, anchor, review status, add buttons, and summary lists etc. by section.